src/test/java/org/openstreetmap/atlas/geography/atlas/items/complex/buildings/BuildingHeightTestCaseRule.java

```diff
@@ -0,0 +1,69 @@
+package org.openstreetmap.atlas.geography.atlas.items.complex.buildings;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.items.complex.Finder;
+import org.openstreetmap.atlas.utilities.collections.Iterables;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+
+/**
+ * Test fixture for the {@link BuildingHeightTestCase}
+ *
+ * @author ajayaswal
+ */
+public class BuildingHeightTestCaseRule extends CoreTestRule
+{
+    public static final long BUILDING_WITH_ZERO_MINHEIGHT_ID = 100000000L;
+    public static final long BUILDING_WITH_NEGATIVE_MINHEIGHT_ID = 200000000L;
+    public static final long BUILDING_WITH_BOTH_HEIGHT_ID = 300000000L;
+    public static final long BUILDING_WITH_NO_MINHEIGHTT_ID = 400000000L;
```

Comment:
Nit: two `TT_ID` --> `T_ID`. Also, consider adding `_` between `MIN` and `HEIGHT` for readability.